src/modules/Elsa.Studio.Workflows.Designer/Components/ActivityWrapper.razor

```diff
@@ -3,20 +3,15 @@
 @using Elsa.Studio.Workflows.Domain.Contexts
 @inherits StudioComponentBase
 @{
-    if (Stats != null)
-    {
-        var color = Stats.Faulted ? Color.Error : Stats.Blocked ? Color.Warning : Stats.Uncompleted > Stats.Completed ? Color.Info : Stats.Completed > 0 ? Color.Success : Color.Default;
-        var content = Stats.Blocked ? null : Stats.Completed.ToString();
-        var icon = content == null ? Stats.Blocked ? Icons.Material.Outlined.HourglassTop : Stats.Uncompleted > 0 ? Icons.Material.Outlined.PlayArrow : null : null;
+    
```

Comment:
Are you deliberately reverting this change or is this a merge issue?

Reply:
this is deliberate; originally I fixed the height issue by reverting the rendering optimization. Now, I am applying the render optimization and fix the height issue by updating the JS file that calculates the activity element's height.